tests/basic/pure_test.go

```diff
@@ -72,7 +76,28 @@ var _ = Describe("{PureVolumeCRUDWithSDK}", func() {
 		for i := 0; i < Inst().GlobalScaleFactor; i++ {
 			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("purevolumestest-%d", i))...)
 		}
-		ValidateApplicationsPureSDK(contexts)
+		for _, ctx := range contexts {
+			// context for pure volume mount tests
+			if ctx.App.Key == mountTestApp {
+				ValidateContext(ctx)
+				vols, err := Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "Failed to get app %s's volumes", ctx.App.Key)
+				log.Infof("volumes of app %s are %s", ctx.App.Key, vols)
+				for _, v := range vols {
+					attachedNode, err := Inst().V.GetNodeForVolume(v, defaultCommandTimeout, defaultCommandRetry)
+					log.FailOnError(err, "Failed to get app %s's attachednode", ctx.App.Key)
+					// This step performs a basic test making sure Volume mount options are working as expected.
+					Step("validating mount options", func() {
+						err = Inst().V.ValidatePureFaFbMountOptions(v.ID, mountoption, attachedNode)
+						dash.VerifySafely(err, nil, "Testing mount options are properly applied on pure volumes")
+					})
+				}
+			} else {
```

Comment:
Can we make this always run the `ValidateContextForPureVolumesSDK` part? That way the mount test is an additional test but we also still validate everything else works (like does `nosuid` break certain functionality)?

Reply:
Done, updated code to do same.